tensorflow/tools/ci_build/ci_sanity.sh

```diff
@@ -129,6 +129,12 @@ do_pylint() {
   echo ""
   echo "check whether pylint is available or not."
   echo ""
+
+  # !!!!!!This is only an hack to workaround CI VM with numpy 1.14.5!!!!!!!!
+  python3.8 -m pip install --upgrade numpy~=1.19.2 lazy-object-proxy
+  python3.8 -m pip show numpy lazy-object-proxy
+
```

Comment:
I've explicitly temp forced numpy for python3.8 here but if you see the log we have still some residual misalignment with the CI VM

```
FAIL: Found 4 errors
tensorflow/python/data/experimental/kernel_tests/data_service_ops_ft_test.py:109: [W0622(redefined-builtin), DataServiceOpsTest.testDispatcherRestartDuringDistributedEpochRepeat] Redefining built-in 'breakpoint'
tensorflow/python/keras/preprocessing/image.py:305: [E0240(inconsistent-mro), DirectoryIterator] Inconsistent method resolution order for class 'DirectoryIterator'
tensorflow/python/ops/control_flow_grad.py:31: [W0622(redefined-builtin), ] Redefining built-in 'exit'
tensorflow/python/ops/control_flow_grad.py:31: [W0622(redefined-builtin), ] Redefining built-in 'tuple'
```